File: Baekjoon/python/1978.py
N = int(input())
num_list = list(map(int, input().split()))

def is_prime(num):
    if num == 1:
        return False
    elif num == 2:
        return True
    for i in range(2, num):
        if num % i == 0:
            return False
    return True

count = 0

for i in num_list:
    if is_prime(i):
        count += 1
print(count)

File: Baekjoon/C++/1978.cpp
#include <iostream>
using namespace std;

int main(){
  int size;
  cin >> size;
  int count = size;

  for (int i = 0 ; i < size; i++){
    int n;
    cin >> n;

    if (n == 1){
      count--;
      continue;
    }

    for (int j = 2 ; j < n - 1; j++){
      if (n % j == 0){
        count--;
        break;
      }
    }
  }
  cout << count;

  return 0;
}
